SF iOS/SF iOS/EventDetails/EventDetailsViewController.m

```diff
@@ -160,5 +167,20 @@ - (void)getTravelTimes {
     }];
 }
 
+// MARK: Share
+
+- (void)share {
+    // TODO: Use the group name and coffeecoffeecoffee.coffee url.
+    NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
+    [formatter setDateStyle:NSDateFormatterMediumStyle];
+    [formatter setTimeStyle:NSDateFormatterShortStyle];
+    
+    NSString *text = [NSString stringWithFormat:@"Join us at %@ for coffee, kicking off %@.", self.event.venueName, [formatter stringFromDate:self.event.date]];
```

Comment:
Would it be beneficial to use the notification copy with the random seed through this API? https://github.com/ay8s/sf-ios/blob/task/event-share/SF%20iOS/SF%20iOS/Helpers/UserNotification/NSString%2BEventNotifications.h#L14